content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst

```diff
@@ -2,41 +2,49 @@
 Incoming shipments and delivery orders
 ======================================
 
-There are several ways to handle how a warehouse receives products (receipts) and ships products
-(deliveries). Depending on several factors, such as the type of products stocked and sold, warehouse
-size, and the amount of daily confirmed receipts and delivery orders, the way products are handled
-when coming in and out of the warehouse can vary a lot. Different settings can be configured for
-receipts and deliveries; they do not have to be configured to have the same number of steps.
+In Odoo, there are many different ways to process receipts and deliveries for products moving in and
+out of a warehouse.
+
+A company's preferred method of receiving or shipping products depends on several factors, such as
+the type of products stocked and sold, the size of the warehouse, the amount of receipts and
+delivery orders processed daily, and more.
+
+Enabling the *Multi-Step Routes* feature in Odoo's *Inventory* app settings allows users to change
+the manner in which receipts and deliveries are processed in the database.
 
 .. seealso::
    - `Using Routes (eLearning Tutorial) <https://www.odoo.com/slides/slide/using-routes-1018>`_
    - `Push & Pull Rules (eLearning Tutorial)
-     <https://www.odoo.com/slides/slide/push-pull-rules-1024>`_
+     <https://www.odoo.com/slides/slide/push-pull-rules-5789>`_
+
+One, two, and three-step receipts and deliveries
+================================================
```

Comment:
Nitpick, but this doc only has one H2 (======)
![image](https://github.com/odoo/documentation/assets/23611711/d6af0a89-ad32-4e89-b73c-7c08e86b5a71)

For users to tell at a glance what info is in this doc, what do you think about removing the "One, two, and three-step receipts and deliveries" section title, and changing the H3 ones below (One-step flow, Two-step flow, etc) into H2?

Reply:
Good idea! Suggestion implemented ✅